### ISSUE TITLE
NoSuchMethodError: android.widget.EditText.setElegantTextHeight

### ISSUE THREAD

**Issue by Kahtaf** [michael-rapp/AndroidMaterialValidation#3]
Hi, I'm seeing this in Crashlytics. This crash primarily happens on Samsung devices. Is it possible to wrap it in a try/catch before calling setElegantTextHeight() on the view? 

```
Caused by java.lang.NoSuchMethodError: android.widget.EditText.setElegantTextHeight
de.mrapp.android.validation.EditText.setElegantTextHeight (EditText.java:931)
de.mrapp.android.validation.EditText.obtainEditTextStyledAttributes (EditText.java:285)
de.mrapp.android.validation.EditText.obtainStyledAttributes (EditText.java:220)
de.mrapp.android.validation.EditText.initialize (EditText.java:204)
de.mrapp.android.validation.EditText.<init> (EditText.java:577)
java.lang.reflect.Constructor.constructNative (Constructor.java)
java.lang.reflect.Constructor.newInstance (Constructor.java:423)
android.view.LayoutInflater.createView (LayoutInflater.java:600)
android.view.LayoutInflater.createViewFromTag (LayoutInflater.java:702)
android.view.LayoutInflater.inflate (LayoutInflater.java:470)
android.view.LayoutInflater.inflate (LayoutInflater.java:398)
android.view.LayoutInflater.inflate (LayoutInflater.java:354)
android.view.View.inflate (View.java:18465)
de.mrapp.android.preference.EditTextPreference.onPrepareDialog (EditTextPreference.java:346)
de.mrapp.android.preference.DialogPreference.showDialog (DialogPreference.java:1001)
de.mrapp.android.preference.DialogPreference.onClick (DialogPreference.java:2161)
android.preference.Preference.performClick (Preference.java:1052)
android.preference.PreferenceScreen.onItemClick (PreferenceScreen.java:229)
android.widget.AdapterView.performItemClick (AdapterView.java:308)
android.widget.AbsListView.performItemClick (AbsListView.java:1510)
android.widget.AbsListView$PerformClick.run (AbsListView.java:3472)
android.widget.AbsListView$3.run (AbsListView.java:4835)
android.os.Handler.handleCallback (Handler.java:733)
android.os.Handler.dispatchMessage (Handler.java:95)
android.os.Looper.loop (Looper.java:146)
android.app.ActivityThread.main (ActivityThread.java:5598)
java.lang.reflect.Method.invokeNative (Method.java)
java.lang.reflect.Method.invoke (Method.java:515)
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1283)
com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1099)
dalvik.system.NativeStart.main (NativeStart.java)
```

Crashlytics captured this issue for a whole bunch of Samsung devices running API levels 17-19.

**Comment by michael-rapp** [comment]
@Kahtaf The source of this issue definitely is the library [AndroidMaterialValidation](https://github.com/michael-rapp/AndroidMaterialValidation), which is used by this project. You have opened [this](https://github.com/michael-rapp/AndroidMaterialValidation/issues/3) issue for the other library. Both problems seem to have the same root cause. Do you agree?

**Comment by michael-rapp** [comment]
Fixed in version  2.1.4.

**Comment by Kahtaf** [comment]
@michael-rapp thanks for the fast response and fix!
